fast/stages/0-org-setup/datasets/hardened/organization/observability/storageIamChanges.yaml

```diff
@@ -0,0 +1,76 @@
+# Copyright 2025 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+alerts:
+  storageIamChanges:
+    combiner: OR
+    conditions:
+    - condition_threshold:
+        aggregations:
+        - alignment_period: 60s
+          cross_series_reducer: REDUCE_SUM
+          group_by_fields:
+          - metric.label.principal
+          - metric.label.method_name
+          - metric.label.project_id
+          - metric.label.location
+          - metric.label.bucket_name
+          per_series_aligner: ALIGN_SUM
+        comparison: COMPARISON_GT
+        duration: 0s
+        filter: resource.type = "logging_bucket" AND metric.type = "logging.googleapis.com/user/storageIamChanges"
+        threshold_value: 0
+        trigger:
+          count: 1
+      display_name: Cloud Storage IAM Permission Changes
+    display_name: Cloud Storage IAM Permission Changes
+    documentation:
+      content: 'Log-based alerting policy in project ${project} detected Cloud Storage
+        Bucket IAM changes.
+
+        This alert helps ensure security by monitoring IAM permission changes to Cloud
+        Storage buckets. ``` resource.type="gcs_bucket" AND protoPayload.methodName="storage.setIamPermissions"
+        ```'
+      mime_type: text/markdown
+logging_metrics:
+  storageIamChanges:
+    bucket_name: log-0/audit-logs
+    description: Cloud Storage IAM Permission Changes
+    filter: resource.type="gcs_bucket" AND protoPayload.methodName="storage.setIamPermissions"
```

Comment:
Is this only for bucket-level changes?
If yes, would it make sense to extend it to object level?